recipes/brotli/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "0.6.pre" %}
+{% set commit = "5db62dcc9d386579609540cdf8869e95ad334bbd" %}
```

Comment:
This is fine and I've done it before myself, but it is nice to have some comments to why you went this route instead of a stable release.